src/metabase/core.clj

```diff
@@ -58,6 +58,7 @@
   []
   (log/info (trs "Starting Metabase version {0} ..." config/mb-version-string))
   (log/info (trs "System info:\n {0}" (u/pprint-to-str (troubleshooting/system-info))))
+  (log/info (trs "Metabase info:\n {0}" (u/pprint-to-str (troubleshooting/metabase-info))))
```

Comment:
This is actualyl causing Metabase to not start because it requires the db be set up

```
java.lang.Exception: DB is not set up. Make sure to call set-default-db-connection! or bind *db-connection*.
```

Just remove it. Sorry.

Reply:
whoops! that's what I get for checking from the REPL.